### ISSUE TITLE
what about the filtered cells?

### ISSUE THREAD

**Issue by WenxuLiu** [AntonioDeFalco/SCEVAN#26]
i have run this pipe in intratumoral model,  and i got a result with 63 filtered cells and 5361 tumor cells.  
is that means, in my data, there is no normal cells?  
and  what about the filtered cells meaning? low quality or something else?
expecting your reply 
thanks very much~

**Comment by WenxuLiu** [comment]
and if  my data have no normal cells,  could i use this pipe to identify that all the  cells are tumor?

**Comment by AntonioDeFalco** [comment]
Hi @WenxuLiu,

> i have run this pipe in intratumoral model, and i got a result with 63 filtered cells and 5361 tumor cells. is that means, in my data, there is no normal cells? and what about the filtered cells meaning? low quality or something else? expecting your reply thanks very much~

yes, filtered cells are low quality cells that are filtered and not used in the pipeline. From the result you got, it looks like your sample is pure and only has tumor cells.

> and if my data have no normal cells, could i use this pipe to identify that all the cells are tumor?

SCEVAN also works with this type of sample by removing a synthetic baseline from the sample and then running the entire analysis.

Alternatively, if the existence of normal cells in the matrix is known, it can be specified with the norm_cell parameter.

I hope I have clarified your doubts. Thanks.

Regards.

**Comment by WenxuLiu** [comment]
> 

thanks for your prompt reply,
I think I totally get it now.

**Comment by Payne-Wang** [comment]
@AntonioDeFalco Hi, thanks for this excellent tool! We encountered a similar issue with this. We used this tool to distinguish the tumor cells in the scRNAseq data from primary tumor tissues and lymph node tissues. But we found that there were only "filtered" and "tumor" epithelial cells (predicted by this tool) in all primary tumor tissues. Did this mean there are no normal epithelial cells in these tissues? On the other hand, in lymph node tissues,there were a large number of immune cells recognized as tumor cells. We are so confused by these results. Any suggestions will be helpful! Thanks for your time!
